wp-includes/class-wp-web-app-manifest.php

```diff
@@ -93,6 +93,9 @@ public function get_theme_color() {
 		if ( current_theme_supports( 'theme-color' ) ) {
 			$theme_color = get_theme_support( 'theme-color' );
 			if ( $theme_color ) {
+				if ( is_array( $theme_color ) && ! empty( $theme_color[0] ) ) {
```

Comment:
I think `! empty( $theme_color[0] )` should be removed here, and the line below can just do:
```php
return array_shift( $theme_color );
```
This will ensure that an array is never returned from this method.

Reply:
You're right. Pushed another commit :+1: